src/module.cpp

```diff
@@ -870,12 +870,11 @@ module::print_py(std::ostream& os,
             if(ins->name() == "@literal")
             {
                 os << mname << ".add_literal(";
-                bool use_abs = false;
-                ins->get_literal().visit([&](auto v) {
-                    use_abs = std::none_of(v.begin(), v.end(), [](auto x) { return x < 0; });
-                });
+                const bool use_abs = false;
                 // Disable abs for now
-                use_abs = false;
+                // ins->get_literal().visit([&](auto v) {
+                //     use_abs = std::none_of(v.begin(), v.end(), [](auto x) { return x < 0; });
```

Comment:
Why is it commented out ?

Reply:
It needs to be implemented. Cppcheck was complaining about dead code here, so I just commented it out for now.

Reply:
I don't understand how is it deadcode ? It is going over literal and trying to set `use_abs` accordingly and would always execute.

Reply:
`use_abs` is assigned `false` and is never assigned `true`. We need to add an API to python for this.